src/Frame.cpp

```diff
@@ -430,7 +430,11 @@ std::unique_ptr<folly::IOBuf> Frame_SETUP::serializeOut() {
   appender.writeBE(static_cast<uint32_t>(version_));
   appender.writeBE(static_cast<uint32_t>(keepaliveTime_));
   appender.writeBE(static_cast<uint32_t>(maxLifetime_));
-  appender.push((const uint8_t*)token_.data().data(), token_.data().size());
```

Comment:
please fix StandardReactiveSocket::clientConnect method to set the flag FrameFlags_RESUME_ENABLE if ConnectionSetupPayload.resumable == true. Otherwise we never send the Token in the setup frame.